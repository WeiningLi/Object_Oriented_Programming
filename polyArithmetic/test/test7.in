create a 1/1 10 2/1 9 3/1 8 4/1 7 1/1 6 1/2 5 28/-4 4 10/5 3 -9/81 2 10/1 1 12/1 0
print a 
create b 1/5 5 10/2 4 1/5 2 1/4 1 4/7 0
print b
create c 0/1 0
print c
subtract a b
add a b
add a c
add c a
subtract a c
subtract c a
add a a
subtruct a a
multiply a a
divide a b
divide b a
quit
