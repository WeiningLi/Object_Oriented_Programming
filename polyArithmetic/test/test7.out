(1)x^10 + (2)x^9 + (3)x^8 + (4)x^7 + (1)x^6 + (1/2)x^5 + (-7)x^4 + (2)x^3 + (-1/9)x^2 + (10)x + (12)
(1/5)x^5 + (5)x^4 + (1/5)x^2 + (1/4)x + (4/7)
0
(1)x^10 + (2)x^9 + (3)x^8 + (4)x^7 + (1)x^6 + (3/10)x^5 + (-12)x^4 + (2)x^3 + (-14/45)x^2 + (39/4)x + (80/7)
(1)x^10 + (2)x^9 + (3)x^8 + (4)x^7 + (1)x^6 + (7/10)x^5 + (-2)x^4 + (2)x^3 + (4/45)x^2 + (41/4)x + (88/7)
(1)x^10 + (2)x^9 + (3)x^8 + (4)x^7 + (1)x^6 + (1/2)x^5 + (-7)x^4 + (2)x^3 + (-1/9)x^2 + (10)x + (12)
(1)x^10 + (2)x^9 + (3)x^8 + (4)x^7 + (1)x^6 + (1/2)x^5 + (-7)x^4 + (2)x^3 + (-1/9)x^2 + (10)x + (12)
(1)x^10 + (2)x^9 + (3)x^8 + (4)x^7 + (1)x^6 + (1/2)x^5 + (-7)x^4 + (2)x^3 + (-1/9)x^2 + (10)x + (12)
(-1)x^10 + (-2)x^9 + (-3)x^8 + (-4)x^7 + (-1)x^6 + (-1/2)x^5 + (7)x^4 + (-2)x^3 + (1/9)x^2 + (-10)x + (-12)
(2)x^10 + (4)x^9 + (6)x^8 + (8)x^7 + (2)x^6 + (1)x^5 + (-14)x^4 + (4)x^3 + (-2/9)x^2 + (20)x + (24)
(1)x^20 + (4)x^19 + (10)x^18 + (20)x^17 + (27)x^16 + (29)x^15 + (10)x^14 + (-13)x^13 + (-263/9)x^12 + (-211/9)x^11 + (787/12)x^10 + (937/9)x^9 + (1825/9)x^8 + (791/9)x^7 + (356/9)x^6 + (-1156/9)x^5 + (-10367/81)x^4 + (412/9)x^3 + (292/3)x^2 + (240)x + (144)
Quotient: (5)x^5 + (-115)x^4 + (2890)x^3 + (-72235)x^2 + (7223955/4)x + (-632134675/14)
Remainder: (325613950/49)x^4 + (-2413519/7)x^3 + (33100937/336)x^2 + (-274562131/392)x + (-883213710/49)
Quotient: 0
Remainder: (1/5)x^5 + (5)x^4 + (1/5)x^2 + (1/4)x + (4/7)
